app/views/apps/index.html.haml

```diff
@@ -1,11 +1,14 @@
 - content_for :title, t('.title')
 - content_for :action_bar do
-  %span= link_to(t('.new_app'), new_app_path, :class => 'add') if current_user.admin?
+  - if current_user.admin?
+    = link_to new_app_path, :class => 'btn btn-success' do
+      %i.glyphicon.glyphicon-plus-sign
+      %span= t('.new_app')
 
-%table.apps
+%table.table-striped.table-hover.apps
```

Comment:
Missing .table class, on bigger resolution table is not resized in whole screen...